src/app/api/entry/[_id]/responses/route.ts

```diff
@@ -0,0 +1,23 @@
+import { NextRequest, NextResponse } from "next/server";
+import connectDB from "@database/db";
+import entrySchema from "@database/volunteerEntrySchema";
+
+type IParams = {
+  params: {
+    _id: string;
+  };
+};
+
+// GET all VolunteerRoles for a specific volunteer entry
```

Comment:
Update this comment (gets all responses not roles)

Reply:
The error message here should also be changed to responses.